client/js/components/procedure/admin/InstitutionTagManagement/InstitutionList.vue

```diff
@@ -162,14 +202,41 @@ export default {
           createdDate: attributes.createdDate.date,
           edit: this.editingInstitutionId === id,
           id,
-          institution: attributes.name,
-          tags: relationships.assignedTags.data
+          name: attributes.name,
+          tags: relationships.assignedTags.data.map(tag => {
+            const tagDetails = this.getTagById(tag.id)
+            return {
+              id: tag.id,
+              type: tag.type,
+              name: tagDetails.name,
+              category: tagDetails.category
+            }
+          })
+        }
+      })
+    },
+
+    selectableColumns () {
+      return this.categoryFieldsAvailable.map(headerField => ([headerField.field, headerField.label]))
+    },
+
+    tagList () {
+      return Object.values(this.institutionTagList).map(tag => {
+        const { id, attributes, relationships } = tag
+        return {
+          id,
+          name: attributes.name,
+          category: relationships?.category?.data
         }
       })
     }
   },
 
   methods: {
+    ...mapActions('InstitutionTagCategory', {
+      listInstitutionTagCategories: 'list'
```

Comment:
i stumbled over list in the name. At least for me it would be more clear if it would be called `fetchInstitutionTagCategories`

Reply:
Done with [cf2d1d8](https://github.com/demos-europe/demosplan-core/pull/4115/commits/cf2d1d8c0f2ec3e2b9e61d9da76c2d787f0e2b36)